sdk/keyvault/keyvault-keys/src/keyVaultCryptographyClient.ts

```diff
@@ -133,20 +134,20 @@ export class KeyVaultCryptographyClient {
    * @param options - Additional options.
    */
   public async encrypt(
-    algorithm: EncryptionAlgorithm,
-    plaintext: Uint8Array,
+    encryptParameters: EncryptParameters,
     options: EncryptOptions = {}
   ): Promise<EncryptResult> {
-    const localCryptographyClient = await this.getLocalCryptographyClient();
-    const requestOptions = operationOptionsToRequestOptionsBase(options);
+    const { algorithm, plaintext, ...params } = encryptParameters;
+    const requestOptions = { ...operationOptionsToRequestOptionsBase(options), ...params };
```

Comment:
Do we need to add the params to the request options? :o unexpected.

Reply:
Indeed we do! that's where `iv`, `aad, and `tag` go - they are part of the request options in the swagger but are EncryptParameters in our APIs so this is where the two converge

Reply:
Maybe that needs a better name though...